app/src/main/java/com/example/nvdovin/weatherapp/ForecastActivity.java

```diff
@@ -34,25 +36,31 @@ protected void onCreate(Bundle savedInstanceState) {
 
     }
 
+    //check data
```

Comment:
so what   "//check data"  helps you? when you commit changes they should be clear, without any comments( instead of adding comments choose meaningful  name for methods)